iroh-bitswap/src/client.rs

```diff
@@ -84,7 +84,7 @@ impl<S: Store> Client<S> {
         let (notify, mut default_receiver): (
             async_broadcast::Sender<Block>,
             async_broadcast::Receiver<Block>,
-        ) = async_broadcast::broadcast(64);
+        ) = async_broadcast::broadcast(4098);
```

Comment:
please use 4096 (mostly for the insane folks like me that like powers of two)

Reply:
Damn I did to mean degree of 2 but my fast math unit has failed